profile_library/jbl/manufacturer.json

```
@@ -0,0 +1,4 @@
+{
+  "name": "JBL",
```

```suggestion
  "name": "jbl",
```

currently this must be lowercased

---

done